classes/services/PKPSubmissionFileService.inc.php

```diff
@@ -777,7 +777,7 @@ public function getWorkflowStageId($submissionFile) {
 				}
 				$queryDao = DAORegistry::getDAO('QueryDAO'); /* @var $queryDao QueryDAO */
 				$query = $queryDao->getById($note->getAssocId());
-				return $query ? $query->getStageId() : null;
+				return $reviewRound ? $reviewRound->getStageId() : null;
```

Comment:
Based on the original forum post, I think this change should be on line 771 and not here.